test/controllers/polls_controller_test.rb

```diff
@@ -2,7 +2,7 @@
 
 class PollsControllerTest < ActionDispatch::IntegrationTest
   setup do
-    @poll = polls(:one)
+    @poll = polls(:poll)
```

Comment:
Is there a reason why one was changed to poll?

Reply:
it was unclear what :one meant (not knowing it was a poll name). It seemed like it was pulling the first poll from some unknown list. This was also a side effect of us considering adding new fixtures, but deciding it was unneeded. The idea is in the future, we might add fixtures like "invalid_poll" or "invalid_title_poll".

Reply:
Sounds good.